snc-library.sh

```diff
@@ -190,15 +190,15 @@ function renew_certificates() {
 
     # Retry 10 times to make sure kubelet certs are rotated correctly.
     i=0
-    while [ $i -lt 10 ]; do
+    while [ $i -lt 60 ]; do
```

Comment:
The comment above this needs to be updated. Probably best to make it more generic "Loop until the kubelet certs are valid for a month" or such.